server.js

```diff
@@ -75,36 +70,29 @@ require("./config/passport");
       saveUninitialized: true,
     })
   );
-// Middleware
-router.get("/", function (req, res) {
-  res.get("index", {
-    user: req.user,
-  });
-});
 
-module.exports = router; 
+
 
 // app.use(bodyParser.urlencoded({ extended: true }))
 // app.use(logger('dev'));
 app.use(express.static('public'));
-app.use('/', router);
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 
   app.use(passport.initialize());
   app.use(passport.session());
   
 // workout log 
-app.use('/', excerRoute); 
+//app.use('/', excerRoute); 
   //welcome signin/log in page
-  app.use('/', indexRoutes);
+app.use('/', indexRoutes);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7828699)
```suggestion
  app.use("/", indexRoutes);
```